src/components/TimeseriesDataExport/TimeseriesDataExport.tsx

```diff
@@ -33,7 +33,7 @@ type TimeseriesDataExportFormProps = TimeseriesDataExportProps &
 // TODO: Check tree shaking for TimeseriesDataExport component
 const { RangePicker } = DatePicker;
 const CELL_LIMIT = 10000;
-const formatData = 'YYYY-MM-DD_HH:mm:ss';
+const formatData = 'YYYY-MM-DD HH:mm:ss';
```

Comment:
You need to change the format example in UI as well

<img width="624" alt="Screenshot 2020-12-04 at 10 25 27" src="https://user-images.githubusercontent.com/24567091/101140084-22281080-361b-11eb-94d8-34375810bc5c.png">

Reply:
Done 👍